src/ts/ConvivaAnalytics.ts

```diff
@@ -443,12 +443,19 @@ export class ConvivaAnalytics {
 
     // Create a Conviva monitoring session.
     this.convivaVideoAnalytics = Conviva.Analytics.buildVideoAnalytics();
+
+    this.convivaVideoAnalytics.setPlayerInfo({
+      [Conviva.Constants.FRAMEWORK_NAME]: 'bitdash',
```

Comment:
We don't use `bitdash` publicly (anymore). Please change to `Bitmovin Player`.

```suggestion
      [Conviva.Constants.FRAMEWORK_NAME]: 'Bitmovin Player',
```